OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -513,7 +513,7 @@ public bool CanTarget(Actor self, in Target target, List<Actor> othersAtTarget,
 					case TargetType.FrozenActor:
 						return CanTargetActor(self, target, ref modifiers, ref cursor);
 					case TargetType.Terrain:
-						return CanTargetLocation(self, self.World.Map.CellContaining(target.CenterPosition), othersAtTarget, modifiers, ref cursor);
```

Comment:
Does this mean `othersAtTarget` is now unused?

Reply:
It seemed that it already was unused.

Reply:
Oh, you mean in `CanTarget`. Wow, that is all dead code. Removed it.